docker/php/custom.ini

```diff
@@ -0,0 +1,29 @@
+; General settings
+date.timezone = "America/Caracas"
```

Comment:
Hi, @ingluife 
Maybe this time zone can be the default and set up optionally with an external arguments.

Reply:
Ok, perfect. Today i will do it. But default Time Zone should be UTC. Do you agree?